VideoLocker/res/layout/fragment_course_detail.xml

```diff
@@ -43,17 +43,9 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:orientation="vertical"
+            android:layout_marginTop="15dp"
```

Comment:
This LinearLayout is no longer serving any purpose (you can put that margin on the FrameLayout inside)